bin/benchmark-diff.swift

```diff
@@ -88,7 +88,18 @@ enum BenchmarkDiff {
         formatter.numberStyle = .decimal
         formatter.locale = Locale.current
         
-        after.sorted(by: { $0.name < $1.name }).enumerated().forEach { (index, metric) in
+        after.sorted { (lhs, rhs) in
+            // Sort by kind of data and then by name
+            if lhs.name.contains("(bytes)") && !rhs.name.contains("(bytes)") {
```

Comment:
Checking the for the existence of `"(bytes)"` or `"(msec)"` in the title seems a _little_ brittle to me but we already rely on this behavior below (line 106 and 117) for converting from bytes to MB and msec to sec so I think it's okay for now. 

If we need to expand to make this more robust in the future we can update the logic in both places.